feature-matrix.json

```diff
@@ -108,7 +108,8 @@
     "default": "✅",
     "query parameter": "",
     "repeat_count": "✅",
-    "create or update Entities": "✅"
+    "create or update Entities": "✅",
+    "setvalue actions": "✅"
```

Comment:
Initial intent was to frame functionality as end users would understand it. In this case, we don’t expose setvalue directly so we had listed it as “trigger” which is the column used in XLSForm for setvalue with a value change event. We probably also should have listed “dynamic defaults” separately. 

That’s still my preference but we could list the action as long as we also check off trigger!

Reply:
That body element/ question type is exposed as “acknowledge” in XLSForm. We had a lengthy discussion when exposing setvalue on value change about whether there was likely to be confusion and decided for better or worse that “trigger” was the best we could come up with for the new concept and that the only people who would see ambiguity would be those who work on ODK XForms clients. That said, we could roll it all under dynamic defaults. Maybe “dynamic defaults (including trigger column)” or something like that.